tests/Microsoft.Identity.Test.Integration.Win8/Microsoft.Identity.Test.Integration.Win8.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworkNetDesktop461>net472</TargetFrameworkNetDesktop461>
+    <TargetFrameworkNetDesktop461>net48</TargetFrameworkNetDesktop461>
```

Comment:
Please rename the property as well

Reply:
I deliberately did not do that because it imports "platform_and_feature_flags.props", and I'm not sure what side effects it would have.

What do you think? (Other projects have the same "issue", e.g. Unit Test)

Reply:
Ack, let's leave it as is then.